types/urijs/test/urijs-dom.test.ts

```diff
@@ -213,6 +213,35 @@ declare var $: (arg?: any) => JQuery;
     parts.protocol === "mailto";
     parts.urn === true;
 
+    /*
+    Tests for URI.buildQuery()
+    From: https://medialize.github.io/URI.js/docs.html#static-buildQuery
+    */
+    const buildQueryData = {
+      foo: 'bar',
+      hello: ['world', 'mars', 'mars'],
+      bam: '',
+      yup: null,
+      removed: undefined,
+    };
+    test(URI.buildQuery(buildQueryData), 'foo=bar&hello=world&hello=mars&bam=&yup');
+    test(URI.buildQuery(buildQueryData, true), 'foo=bar&hello=world&hello=mars&hello=mars&bam=&yup');
+
+    /*
+    Tests for URI.parseQuery()
+    From: https://medialize.github.io/URI.js/docs.html#static-parseQuery
+    */
+    test(URI.parseQuery('?foo=bar&hello=world&hello=mars&bam=&yup'), {
+      foo: 'bar',
+      hello: ['world', 'mars'],
```

Comment:
I do not understand what are these tests for…are we testing the library in this repository? It does not really make much sense IMO, these files should be eliminated.

Reply:
DefinitelyTyped tests are meant to check whether a type definition works correctly in common use cases.

Reply:
Fair enough, although to test types we should use the `// $ExpectType` assertions. In any case, it is not a big deal.